tests/test_data/vasp/KNbO3_ferroelectric/polarization_interpolation_0/inputs/INCAR

```diff
@@ -0,0 +1,24 @@
+ALGO = Fast
+EDIFF = 1e-05
+ENAUG = 1360
+ENCUT = 680
+GGA = Ps
+ISMEAR = -5
+ISPIN = 2
+KSPACING = 0.22
```

Comment:
We removed KSPACING from the vasp input sets. So you will need to remove this line from all the incars in the `inputs` directories and also add a KPOINTS file in the `inputs` directories to stop the errors.

Reply:
done